arrows/vxl/tests/test_aligned_edge_detection.cxx

```diff
@@ -0,0 +1,94 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
```

Comment:
I wonder if we really want to document source files for tests. I seem to recall the usual reason for a `\file` is that documentation therein may not show up otherwise (although I also want to say that's only the case for e.g. macros and maybe free functions).

Reply:
I agree this doesn't seem to add much.